homeassistant/components/homewizard/entity.py

```diff
@@ -2,13 +2,23 @@
 
 from __future__ import annotations
 
-from homeassistant.const import ATTR_CONNECTIONS, ATTR_IDENTIFIERS
+from homeassistant.const import ATTR_CONNECTIONS, ATTR_IDENTIFIERS, ATTR_MODEL
 from homeassistant.helpers.device_registry import CONNECTION_NETWORK_MAC, DeviceInfo
 from homeassistant.helpers.update_coordinator import CoordinatorEntity
 
 from .const import DOMAIN
 from .coordinator import HWEnergyDeviceUpdateCoordinator
 
+TYPE_MODEL_MAP = {
```

Comment:
Ideally, this would live in the library so other consumers of the lib have this data too. 

Also, just an option, `hass.config.language` can be used to offer translated names.

Reply:
Works for me! Can pick that up in a follow-up!